README.md

```diff
@@ -142,14 +142,22 @@ Execute all PowerShell calls as sudo.
 This is necessary in certain cases, such as when `pwsh` is installed via `snap` and is only available via `sudo` unless you customise the system's configuration.
 
 * `downloads`- map[string, string], defaults to `{"./PesterTestResults.xml" => "./testresults}"`.
-Files to download from SUT to local system, used to download the pester results localy. 
+Files to download from SUT to local system, used to download the pester results localy.
 The key is the remote file (relative to verifier folder or absolute), the value is the directory (ends with / or \\) it should be saved to (relative to pwd or absolute).
   ```yaml
   downloads:
       PesterTestResults.xml: "./output/testResults/"
       kitchen_cmd.ps1: "./output/testResults/"
   ```
 
+* `environment_variables` - map[string, string], defaults to `{}`.
+Environment variables to set in SUT for your pester tests to access.
+  ```yaml
+  environment_varialbes:
```

Comment:
```suggestion
  environment_variables:
```

I'm also looking at this wondering if simply `environment` is enough on its own for the name of the key here?

Reply:
I like the idea of just using environment. I've updated and pushed changes.